benchmark/stats/histogram.go

```diff
@@ -73,16 +76,19 @@ func (v HistogramValue) String() string {
 	return b.String()
 }
 
-// A Histogram accumulates values in the form of a histogram. The type of the
-// values is int64, which is suitable for keeping track of things like RPC
-// latency in milliseconds. New histogram objects should be obtained via the
-// New() function.
+// A histograms is stored with exponentially increasing bucket sizes.
```

Comment:
Histogram accumulates values in the form of a histogram with exponentially increased bucket sizes.